charts/common/values.schema.json

```diff
@@ -619,43 +619,89 @@
                 }
               }
             },
-            "networkpolicy": {
+            "networkpolicies": {
               "type": "object",
-              "required": [
-                "deploy"
-              ],
-              "properties": {
-                "deploy": {
-                  "type": "boolean",
-                  "default": false
-                },
-                "ipBlock": {
+              "patternProperties": {
+                "^.*$": {
                   "type": "object",
-                  "required": [
-                    "cidr"
-                  ],
                   "properties": {
```

Comment:
Are any of the underlying properties required?
If so, specify them.
If not, add sane defaults?

When setting the following configuration in values.yaml
```
...
    networkpolicies:
      networkpolicy-1: {}
```

Is the following output desired?
```
spec:
  podSelector:
    matchLabels:
      app.kubernetes.io/name: common
      app.kubernetes.io/component: component-2
  policyTypes:
```

Reply:
thanks, fixed it as well.